### PR TITLE
Merge branch 'tooling-update' into main

### PR DESCRIPTION
**Pull request by raed-emiketic**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -57,7 +36,7 @@ deploy:
   dependencies:
     - build
   tags:
-    - docker
+    - deploy:advences
```
Comment (mehdimahmoudi) [inline .gitlab-ci.yml]: @emiketic-admin @raed-emiketic This shouldn't be set to Advences

File: deploy.sh [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-
-if [ -z "$TARGET" ] || [ ! -d "$TARGET" ] ; then
+if [ -z "$BUILD" ] || [ ! -d "$BUILD" ] ; then
   exit 1
 fi
 
-if [ -n "$PREVIEW_FTP_HOST" ] && [ "$PREVIEW_FTP_PATH" ] && [ -n "$PREVIEW_FTP_USERNAME" ] && [ -n "$PREVIEW_FTP_PASSWORD" ]; then
-  cd $TARGET
-  ncftpput -mR -v -u "$PREVIEW_FTP_USERNAME" -p "$PREVIEW_FTP_PASSWORD" "$PREVIEW_FTP_HOST" "$PREVIEW_FTP_PATH" ./
+if [ -n "$DEPLOY_PATH" ] && [ -d "$DEPLOY_PATH" ]; then
```
Comment (mehdimahmoudi) [inline deploy.sh]: @emiketic-admin On en a déjà parlé je pense mais j'aime pas trop ce nom de variables. On les avait pourtant retravaillées et discutée ensemble par rapport au starter kit et sa norme en général.
As-tu une argumentation spécifiques qui appuie ton choix de ce nom de variables par rapport à ce que j'avais proposé la dernière fois et qui a été supprimé ici en rouge ?

File: gatsby-config.js [hunk 3]
```diff
@@ -40,10 +43,41 @@ module.exports = {
         name: 'gatsby-starter-default',
         short_name: 'starter',
         start_url: '/',
+        // @TODO: Change with actual background and theme color
         background_color: '#663399',
         theme_color: '#663399',
         display: 'minimal-ui',
-        icon: 'src/images/gatsby-icon.png', // This path is relative to the root of the site.
+        icon: 'src/images/gatsby-icon.png',
```
Comment (mehdimahmoudi) [inline gatsby-config.js:50]: Good @raed-emiketic

File: gatsby-config.js [hunk 4]
```diff
@@ -40,10 +43,41 @@ module.exports = {
...
+      },
+    },
+    {
+      // @TODO: change app ID
```
Comment (mehdimahmoudi) [inline gatsby-config.js:54]: Thanks for the reminder @raed-emiketic ! :)

File: package.json [hunk 5]
```diff
@@ -8,14 +8,8 @@
...
-  "license": "MIT",
-  "homepage": "https://www.eyespot.agency",
-  "keywords": [
-    "gatsby",
-    "react",
-    "gatsby starter",
-    "eyespot"
-  ],
+  "license": "Proprietary",
+  "homepage": "https://www.emiketic.com",
```
Comment (mehdimahmoudi) [inline package.json]: @raed-emiketic You need to switch back to the initial licence and meta data that were removed above. Starter kit is under MIT

File: .gitlab-ci.yml (hunk 1 above)
Comment (raed-emiketic) [reply .gitlab-ci.yml]: I'll fix it now

File: package.json (hunk 5 above)
Comment (raed-emiketic) [reply package.json]: OKay I'll fix it

File: src/pages/sample-page.js [hunk 6]
```diff
@@ -9,6 +9,12 @@ import { Link } from 'gatsby';
 // UI lib components
 import { Container } from 'react-grid-system';
 
+/* ------------------------------ Static Image ------------------------------ */
```
Comment (mehdimahmoudi) [inline src/pages/sample-page.js:12]: @raed-emiketic You should install Comment Divider (which is identified as a recommended extension us under .vscode > .extensions, and visible on the sidebar in VSCode): https://marketplace.visualstudio.com/items?itemName=stackbreak.comment-divider
This way we all have the same comment format

If this already the case, that's fine

File: src/shared/styles/index.scss [hunk 7]
```diff
@@ -1,5 +1,5 @@
 @import './variables.scss';
-@import './theme-gatsby-default.scss';
```
Comment (mehdimahmoudi) [inline src/shared/styles/index.scss]: @raed-emiketic `theme-gatsby-default` should be placed back in the starter kit unlike customer project where we remove it

File: src/pages/sample-page.js (hunk 6 above)
Comment (raed-emiketic) [reply src/pages/sample-page.js:12]: already installed and worked with

**Review by mehdimahmoudi — CHANGES_REQUESTED** [review]
My comments left on the PR

File: deploy.sh (hunk 2 above)
Comment (raed-emiketic) [reply deploy.sh]: @mehdimahmoudi  it's my fault, I copied some files from wensh project without review.
I fixed it

**Pull request merged**